File: 1662-check-if-two-string-arrays-are-equivalent/1662-check-if-two-string-arrays-are-equivalent.java
class Solution {
    public boolean arrayStringsAreEqual(String[] word1, String[] word2) {
        String s1="";
       String s2 = "";
        for(int index = 0;index < word1.length;index++)
        {
    
    
            s1 += word1[index];
        }
        for(int index = 0;index < word2.length;index++)
        {
    
    
            s2 += word2[index];
        }
        return s1.equals(s2);
    }
}

File: 1662-check-if-two-string-arrays-are-equivalent/1662-check-if-two-string-arrays-are-equivalent.js
/**
 * @param {string[]} word1
 * @param {string[]} word2
 * @return {boolean}
 */
var arrayStringsAreEqual = function(word1, word2) {
    let s1=word1.join('');
    let s2=word2.join('');
return s1===s2;
};